Optional update to salt generation at first boot and live stream fixes.

MPTMG opened the pull request:
Here is the change I was talking about. FWIW your method seems like it should work, but was causing it to hang during firstboot.

JedMeister committed:
Merge pull request #1 from MPTMG/master

nginx from repo

mptmg committed:
Merge pull request #3 from JedMeister/pre-release-tweaks

Updating Repo to Include TKL Changes

Commit:
Updates to salt at firstboot

Commit:
Live Stream Directories and DB update to latest version

JedMeister committed:
Merge pull request #2 from JedMeister/pre-release-tweaks

Pre release tweaks

JedMeister committed:
don't cat /dev/urandom - just get what we need

JedMeister committed:
add domain auto redirect

JedMeister committed:
adjust youphptube version in changelog and bump date

mptmg committed:
Add Encoder

Adding YouPHPTube-Encoder For Processing Recordings

mptmg committed:
Add Encoder

Adding YouPHPTube-Encoder For Processing Recordings

mptmg committed:
FixMinor Typos Preventing Build and Config

Indent error python and missing directory error (build).  Enable encoder site by default.

mptmg committed:
FixMinor Typos Preventing Build and Config

Indent error python and missing directory error (build).  Enable encoder site by default.

mptmg committed:
Update To TKL Version For Merge

mptmg committed:
Merge pull request #4 from turnkeylinux-apps/master

Update Master to Turnkey Master

mptmg committed:
Merge pull request #5 from MPTMG/video-record

Video record

JedMeister commented:
Hey @MPTMG - Apologies that I haven't been completely transparent on this... (been squirrelling away in the background).

I've actually already built the v15.0 appliance from TurnKey master's current HEAD: https://github.com/turnkeylinux-apps/youphptube and it's in our build bucket ready to be published (hopefully tonight my time).

I actually built it a couple of days ago, but due to a string of unrelated events (bugs etc in other apps) the publishing has been delayed.

A quick look over the code you've added suggests that you've added encoding which is a cool option. I'm guessing that is perhaps also related to your work re the game streaming stuff? As much as I'd love to add it, I also really want to finalise the last part of v15.0 as it's really dragging.

I do plan to do more regular appliance updates in v15.x though, so we can build v15.1 of YouTubePHP with encoder included; pretty soon hopefully. I hope you're ok with that...

mptmg committed:
Rewrite Rule Breaks YouPHPTube

When Used with and IP Address and not a domain this code breaks YouPHPTube's ability to load.
Commented Out apache config file rewrite details and change during intiital boot.

MPTMG commented:
Finalize 15.0 no need to include current changes now anyway as it was added for the game stream function to build in additional functionality that would be useful.

Just be advised the rewrite rule that was added caused some errors for me trying to load YouPHPTube. As soon as I commented out the rewrite rules added for https(443) in the yphptube.conf file everything worked just fine. Naturally the code to automatically edit that in the inithook needed to be commented as well to have turnkey-init result in a fully functional appliance.

JedMeister committed:
add logo and screenshots

JedMeister committed:
fix readme title typo

JedMeister committed:
fix browser redirect loop if IP set as APP_DOMAIN

JedMeister committed:
Merge branch 'fix-redirect-for-ip' - part of #1230

JedMeister committed:
move Apache port config to conf script

JedMeister committed:
shouldn't need to overwrite apache.conf; adjust vhost configs

JedMeister committed:
bump changelog and tweak readme

mptmg committed:
Remove conf file

mptmg committed:
Delete ports.conf

mptmg committed:
Maunal Update

mptmg committed:
Manual Update For Merge

mptmg committed:
Manual Update

mptmg committed:
Merge pull request #6 from turnkeylinux-apps/mptmg-encoding-addition

Mptmg encoding addition

JedMeister committed:
Merge branch 'mptmg-encoding-addition'

JedMeister committed:
finalise changelog for fixes and improvment

mptmg committed:
Merge pull request #7 from turnkeylinux-apps/master

Sync With TKL Branch

mptmg committed:
Keeping DB Version Up To Date

Database Version now at 6.2. Updating db to reflect current db version. Causes non-critical error when YouPHPTube attempts to update database to 6.2 even though database is already on 6.2 in build.
